.eslintrc.js

```diff
@@ -1,24 +1,41 @@
-const isDistribution = process.env.NODE_ENV === 'dist'
+const isDistribution = process.env.NODE_ENV === "dist"
 
 module.exports = {
-  root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    sourceType: 'module'
-  },
-  env: {
-    browser: true,
-    jasmine: true
-  },
-  globals: {
-    angular: true,
-    inject: true
-  },
-  extends: 'standard',
-  rules: {
-    'arrow-parens': 0,
-    'generator-star-spacing': 0,
-    'no-console': isDistribution ? 2 : 0,
-    'no-debugger': isDistribution ? 2 : 0
-  }
+    "root": true,
+    "parser": "babel-eslint",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "env": {
+        "browser": true,
+        "node": true,
+        "jasmine": true,
+        "es6": true
+    },
+    "globals": {
+        "angular": true,
+        "inject": true
+    },
+    "extends": "ovh",
+    "rules": {
```

Comment:
Why do you need to update theses rules ? 2 solutions : 
+ Updating theses rules is legit, and we need to update the OVH rules
+ Else, we should follow the existing rules :)

But please, don't create a different `.eslintrc.js`. The goal is to homogenise :)

Reply:
OVH rules will be updated and this custom config removed at the end of V2